src/stream_base.cc

```diff
@@ -102,8 +102,7 @@ int StreamBase::Writev(const FunctionCallbackInfo<Value>& args) {
 
   size_t count = chunks->Length() >> 1;
 
-  uv_buf_t bufs_[16];
-  uv_buf_t* bufs = bufs_;
+  MaybeStackBuffer<uv_buf_t> bufs(count);
```

Comment:
Can you use `MaybeStackBuffer<uv_buf_t, 16>`? That should keep the current default size, which seems reasonable to me